src/main/java/net/whistlingfish/harmony/HarmonyClient.java

```diff
@@ -59,7 +61,7 @@
 public class HarmonyClient {
     private static final Logger logger = LoggerFactory.getLogger(HarmonyClient.class);
 
-    public static final int DEFAULT_REPLY_TIMEOUT = 5_000;
+    public static final int DEFAULT_REPLY_TIMEOUT = 30_000;
```

Comment:
So getting the current activity can be blocked if the hub is still starting.  This prevents a timeout happening